### ISSUE TITLE
Linux Endian macros not available on OS X

### ISSUE THREAD

**Issue by mherb** [Phibedy/LMS#77]
LMS currently fails to compile on OS X due to

```
lms/main/internal/profiler.cpp:81:54: error: use of undeclared identifier 'htobe64'
    *reinterpret_cast<int64_t*>(&datagram.data[1]) = htobe64(now.micros());
```

A solution might be to include something like https://gist.github.com/yinyin/2027912 in LMS, as endian conversion might be needed at some other points as well (also in modules/libs/services..).

Adding a `lms::endian` utility class might be worth it.. What do you think @syxolk ?

**Comment by syxolk** [comment]
Probably fixed with 83e22c1fe26655d5b0cfeebc71d711a440db6769.

**Comment by mherb** [comment]
Fixed uint64_t testcase in 6abf6129. Works now on OS X too, thanks alot!
